ci/docker/runtime_functions.sh

```diff
@@ -73,6 +73,28 @@ build_jetson() {
     popd
 }
 
+build_armv6() {
+    set -ex
+    pushd .
+    cd /work/build
+    cmake \
+	-DCMAKE_TOOLCHAIN_FILE=$CROSS_ROOT/Toolchain.cmake \
+        -DUSE_CUDA=OFF \
+        -DUSE_OPENCV=OFF \
+        -DUSE_SIGNAL_HANDLER=ON \
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+        -DUSE_MKL_IF_AVAILABLE=OFF \
+	-DUSE_LAPACK=OFF \
```

Comment:
Could you add a comment describing the LAPACK situation (statically linked with OpenBLAS etc..)?

Reply:
Done